### DISCUSSION TITLE
Plots are 7.4GB larger than with Mad Max Plotter?

### DISCUSSION THREAD

**Discussion by xhamma** [Chia-Network/bladebit#71] [Q&A]
Hello all, so I added more memory to my plotter so I could run bladebit and it does work pretty good (5 minutes faster per plot than Mad Max), however the K32 plots that bladebit turns out are 7.4GB larger than those turned out by Mad Max... So this equates to me loosing density on my storage at a rate of 1 plot lost for each 13.65 plots plotted with this plotter. Is anyone else seeing this behaviour? 108.8GB for bladebit vs 101.4GB for Mad Max on the same plotter and drive.

**Comment by harold-b** [comment]
What is the block size on the drive where to which you are plotting?

**Comment by xhamma** [comment]
The drives all have a 512 Block size. I am plotting to a RAID0 array, it is the same array in both cases.

**Reply by harold-b** [reply]
Is that the block or sector size?
What output do you get when you do a `stat <some_file_in_plotting_drive>`?

Either way The only dead space in a plot is padding to round up to block size, which for most users will be 4096 bytes. Totaling to at most 40KiB of padding, not much. If yours is indeed 512, it would come to even less. The rest of the space occupied is **valid plot data**, so I am not sure what you mean by losing density.

As for the size discrepancy itself, do you have a full log output from said plot? Also, would you mind sharing a plot id for one of those plots?
